src/odemis/util/test/comp_test.py

```diff
@@ -247,6 +247,22 @@ def test_clipping_axis_range(self):
         finally:
             self.focus.updateMetadata({model.MD_POS_ACTIVE_RANGE: {}})
 
+    def test_focus_outside_range(self):
+        # focus position above the absolute range
+        focus_range = self.focus.axes["z"].range
+        self.focus.moveAbsSync({"z": focus_range[1] + 100e-6})
```

Comment:
This is a good test, but confusing as to why it works. Normally the actuator should fail if a move is requested outside the range.
What happens is that the "focus" component hasn't been referenced. In such case, Odemis is a little loose in the checks, and allows going up to rng * 2. That's why this move is still allowed.
=> Please add this info in comment, to explain the magic of this code :-)